cni-plugin/pkg/install/install.go

```diff
@@ -118,6 +120,20 @@ func loadConfig() config {
 	return c
 }
 
+// getServiceAccountFilePaths returns the mount paths for a container
+// In the case of Windows HostProcess containers this prepends the CONTAINER_SANDBOX_MOUNT_POINT env variable
+// for other operating systems or if the sandbox env variable is not set it returns the standard mount points
+// see https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#volume-mounts
+func getHostPath(path string) string {
```

Comment:
if you only support Containerd  1.7+ this shouldn't be needed and would clean up this section of code

Reply:
I see, so then just referring to the paths as `/host/foo` would work on Windows without any need for change? Thanks, I hadn't tried that

Reply:
yes, if they are volumemouns to the `/host/` location that should work.  This is why the In-Cluster config now works 🚀

Reply:
@jsturtevant I got rid of this `getHostPath()` function on my latest commit, though I still needed to use the `$CONTAINER_SANDBOX_MOUNT_POINT` env var when referring to the binaries inside the container, which from what I read on https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#containerd-v1-7-and-greater shouldn't be needed, right? Will tag you on the relevant chunk of code...

Reply:
Just a quick update: I ended up implementing support for containerd v1.6, so we're using `$CONTAINER_SANDBOX_MOUNT_POINT` just about everywhere that's necessary. Everything's working on my tests so far, and that way we can support a containerd version that's widely adopted right now...